client/src/components/Assets.tsx

```diff
@@ -70,9 +72,9 @@ export const Assets = () => {
         </div>
 
         {transformCoinsToAssets(coins).map((coinData) => (
-          <div className={style.transferPropContainer} key={coinData.color}>
+          <div className={style.transferPropContainer} key={coinData.assetId}>
             <div className={style.transferPropInput}>
-              <span>{coinData.amount}</span>
+              <span>{formatEther(coinData.amount)}</span>
```

Comment:
Is this actually okay? We won't be using 18 decimals, we'll be using 9 at most.

Reply:
Do we have a spec for this? If the max is 9, I can create a very similar utility.

Reply:
There's no spec for decimals on fungible tokens yet. But just based on the fact that we use `u64` there's no way to handle 18 decimals.

Reply:
Are we waiting for the above issue to be merged?

Reply:
Just push a temp fix for it